plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/deployimage/DeployImageWizardModel.java

```diff
@@ -463,17 +470,34 @@ public void setDockerConnection(IDockerConnection dockerConnection) {
 	}
 
 	@Override
-	public boolean imageExists(String image) {
-		if (dockerConnection == null || StringUtils.isBlank(image)) {
+	public boolean imageExistsLocally(String imageName) {
+		if (dockerConnection == null || StringUtils.isBlank(imageName)) {
 			return false;
 		}
-		DockerImageURI imageName = new DockerImageURI(image);
-		String repo =  imageName.getUriWithoutTag();
-		String tag = StringUtils.defaultIfBlank(imageName.getTag(),"latest");
+		DockerImageURI imageURI = new DockerImageURI(imageName);
+		String repo =  imageURI.getUriWithoutTag();
+		String tag = StringUtils.defaultIfBlank(imageURI.getTag(),"latest");
```

Comment:
@adietish please see https://github.com/openshift/openshift-restclient-java/pull/121 for making the 'LATEST' constant public and reusable.